advanced/Scripts/version.sh

```diff
@@ -140,6 +148,11 @@ getLocalBranch(){
 }
 
 versionOutput() {
+    [[ "$1" == "AdminLTE" ]] && [[ "${INSTALL_WEB_INTERFACE}" != true ]] && {
+        echo "  WebAdmin not installed"
+        return 1
+    }
+
```

Comment:
```suggestion
    if [[ "$1" == "AdminLTE" && "${INSTALL_WEB_INTERFACE}" != true ]]; then
        echo "  WebAdmin not installed"
        return 1
    fi
```
Mostly to comply with coding standards and to not have an >0 "exit" code when the conditions are not met (e.g. to allow `set -e` in the future)🙂.